yarn-project/cli/src/cmds/add_contract.ts

```diff
@@ -31,7 +30,7 @@ export async function addContract(
     initializationHash,
     contractClassId: getContractClassFromArtifact(artifact).id,
     portalContractAddress: portalContract ?? EthAddress.ZERO,
-    publicKeysHash: computePublicKeysHash(publicKey),
+    publicKeysHash: publicKeysHash ?? Fr.ZERO,
```

Comment:
If we have the zero values. We will have an odd case in the circuits where we would first read the registry, see zeros, and then read the pre-image which is also zeros and then fail later, or need to skip even trying to do the request? We should probably discuss this when creating the full "api"

Reply:
Good point. Will create an issue and link it there.